README.md

```diff
@@ -49,4 +49,12 @@ wot-servient
 * go to http://localhost:8080/counter and you'll find a thing description.
 * you can query the count by http://localhost:8080/counter/properties/count
 * you can modify the count via POST on http://localhost:8080/counter/actions/increment and http://localhost:8080/counter/actions/decrement
-* application logic is in ``examples/scripts/counter.js``
\ No newline at end of file
+* application logic is in ``examples/scripts/counter.js``
+
+## How to use the library
+
+This library implements the Scripting API defined in the [WoT Current Practices](https://w3c.github.io/wot/current-practices/wot-practices.html#scripting-api) document. 
+
```

Comment:
We should also point to https://w3c.github.io/wot-scripting-api/ - but can do that when we merge tthe `new-api` branch